packages/app/src/systems/SignUp/machines/signUpMachine.ts

```diff
@@ -28,6 +28,8 @@ export enum SignUpType {
 type FormValues = {
   mnemonic?: string[];
   password?: string;
+  mnemonicConfirmation?: string[];
+  positionsForConfirmation?: number[];
 };
```

Comment:
We should not have the mnemonic and password on the context. Can we remove it? Any technical blockers here?

Reply:
we need the `mnemonic` on the state for the MnemonicConfirm component, on the first flow before a reload case.

but we can remove the password.

Reply:
@luizstacio 

To create the Vault, we need two inputs `mnemonic` and `password`, that are set at different steps / components.

I tried removing the `password` from the context.

It works for the create wallet flow because the when user sets the password on the first step, we generate the mnemonic, and we can create the vault since we have both inputs in one step.

but it doesn't work for the recovery flow, because the Vault is created at the end of the flow, and we would need the password, and the user entered mnemonic, we need to store the password in the context while the user enters the mnemonic.